media/css/m24/components/footer-refresh.scss

```diff
@@ -173,8 +196,8 @@ $max-footer-content-width: 1440px;
 
             select {
```

Comment:
**Issue/suggestion:** It's impossible to distinguish that these fields are select boxes, as they appear identical to a text input? Can we add some form is chevron perhaps to help identify their function?

![image](https://github.com/user-attachments/assets/f12bfb06-c346-446f-a21a-cadd3d25f1a7)

Reply:
**issue:** both the email field and the select inputs could also benefit from some more easily identifiable focus states?

Reply:
Tracking the accessibility issues in https://github.com/mozilla/bedrock/issues/14830#issuecomment-2345240527 for now, though they might need to be broken out into separate issues.